docs/source/tutorials/structure_your_input.rst

```diff
@@ -161,11 +107,187 @@ An example for an input directory with videos could look like this:
         +-- my_video_3.avi
         +-- my_video_4.avi
 
-We assign a weak label to each video.
+.. code-block:: python
+
+   from lightly.data import LightlyDataset
+
+   video_dataset = LightlyDataset(input_dir='video_dir/')
+
+The dataset assigns each video frame its video as label.
+
+.. note::
+
+   To use video-specific features of Lightly SSL download the necessary extra dependencies `pip install lightly"[video]"`. Furthermore,
+   randomly accessing video frames is slower compared to accessing the extracted frames on disk. However,
+   by working directly on video files, one can save a lot of disk space because the frames do not have to
+   be extracted beforehand.
+
+PyTorch Datasets
+----------------
+
+You can also use native `torchvision <https://pytorch.org/vision/main/datasets.html>`_ datasets with Lightly SSL directly.
+Just create a dataset as you normally would and apply transforms for greater control over the dataloader. For example, BYOL
+(Bootstrap your own element) expects two views of an input image. To achieve this, we can use the `BYOLTransform`
+while creating the dataset instance, which will lead to the dataloader returning two views per batch, viz.
+
+.. code-block:: python
+
+   import torchvision
+   from lightly.transforms.byol_transform import (
+       BYOLTransform,
+       BYOLView1Transform,
+       BYOLView2Transform,
+   )
+
+   transform = BYOLTransform(
+       view_1_transform=BYOLView1Transform(input_size=32, gaussian_blur=0.0),
+       view_2_transform=BYOLView2Transform(input_size=32, gaussian_blur=0.0),
+   )
```

Comment:
BYOL is maybe not a great example as it requires more boilerplate code for creating the transformer compared to SimCLR. Consider switching to SimCLR